dds_web/security/auth.py

```diff
@@ -112,17 +131,20 @@ def __verify_general_token(token):
     if expiration_time and (
         dds_web.utils.current_time() <= datetime.datetime.fromtimestamp(expiration_time)
     ):
-        username = data.get("sub")
-        if username:
-            user = models.User.query.get(username)
-        if user and user.is_active:
-            return user, data
-
-        return None, data
+        return data
 
     raise AuthenticationError(message="Expired token")
 
 
+def user_from_subject(subject):
```

Comment:
can we change the access modifier of this to private and update the calls (only in this file)?

Reply:
Yes! Done!